pkg/vsphere/disk/disk_manager_test.go

```diff
@@ -369,3 +371,179 @@ func TestRefCounting(t *testing.T) {
 		return
 	}
 }
+
+func TestInUse(t *testing.T) {
```

Comment:
Possibly have a test for a disk that is attached to two vms. One on and one off.